docs/dev/kube-apiserver-health-check.md

```diff
@@ -0,0 +1,70 @@
+## Graceful Termination
+`kube-apiserver` in OpenShift is fronted by an external and an internal Load Balancer. This document serves as a
+guideline on how to properly configure the health check probe of the load balancer so that when a `kube-apiserver`
+instance restarts we can ensure:
+- The load balancer detects it and takes it out of service in time. No new request should be forwarded to the 
+  `kube-apiserver` instance when it has stopped listening.
+- Existing connections are not cut off hard, they are allowed to complete gracefully.
```

Comment:
up to 60s

Reply:
fixed

Reply:
not fixed in this line, which is fine, since this is the summary.  But the details below talk about 60s, so marking this thread resolved.